tests/test_filter_abund.py

```diff
@@ -228,16 +228,18 @@ def test_filter_abund_4_retain_low_abund():
 
     counting_ht = _make_counting(infile, K=17)
 
-    script = 'filter-abund.py'
-    args = ['-V', counting_ht, infile]
-    utils.runscript(script, args, in_dir)
+    for script, args in (('filter-abund.py',
```

Comment:
could you split this into two tests, please? we try to keep the tests as free of for loops and if statements as possible (i.e. in declarative mode) for simplicity purposes.